cmd/xl-v1-multipart.go

```diff
@@ -624,7 +624,7 @@ func (xl xlObjects) ListObjectParts(bucket, object, uploadID string, partNumberM
 	nsMutex.Lock(minioMetaBucket, pathJoin(mpartMetaPrefix, bucket, object, uploadID), opsID)
 	defer nsMutex.Unlock(minioMetaBucket, pathJoin(mpartMetaPrefix, bucket, object, uploadID), opsID)
 
-	if !xl.isUploadIDExists(bucket, object, uploadID) {
+	if !xl.isUploadIDExists(bucket, object, uploadID, false) {
```

Comment:
if this is the only place where it passes `false` and hence an exception then we can see if can make the code simpler by always having valid-quorum for isUploadIDExists() so that we don't have two variants of with and without quorum.

If we do with-quorum-always:
Upside is simpler code (i.e just one variant of the function)
Downside is ListObjectParts is slightly slower.

So doing with-quorum-always looks a better choice more so if we assume that reuploads and hence ListObjectParts calls are rare. Another thing is ListObjectParts calls are rare compared to PutObjectPart where we do with-quorum-always, hence even if there is performance problem because of with-quorum-always check then it will hit PutObjectPart rather than ListObjectParts.